test/support/telemetry/test_helpers.ex

```diff
@@ -0,0 +1,41 @@
+defmodule Telemetry.TestHelpers do
```

Comment:
Why does this module have different naming than the app?
Telemetry vs PrometheusTelemetry

Reply:
It is pretty useful outside of this project, so I was considering making a lib, though it would be an admittedly small lib.